charts/datasaur/templates/config/metabase-importer-secret.yaml

```diff
@@ -0,0 +1,17 @@
+{{- if .Values.metabaseImporter.enabled  }}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ .Release.Name }}-metabase-importer-secret
+type: Opaque
+stringData:
+  AWS_ACCESS_KEY: {{ required "AWS_ACCESS_KEY is required" .Values.metabaseImporter.secret.aws.access_key | quote }}
+  AWS_SECRET_KEY: {{ required "AWS_SECRET_KEY is required" .Values.metabaseImporter.secret.aws.secret_key | quote }}
+  AWS_REGION: {{ required "AWS_REGION is required" .Values.metabaseImporter.secret.aws.region | quote }}
```

Comment:
as discussed yesterday, please add if **values.analytics.enable** and ENABLE_GLAIR_ANALYTICS=true and ENABLE_NEW_ANALYTICS=true

in datasaur configmap